app/routes/searchRoomCommercial.tsx

```diff
@@ -0,0 +1,54 @@
+import { Card, CardGroup, CardHeader, Header, Label, Link, Select } from "@trussworks/react-uswds";
+import Breadcrumbs from "~/components/Breadcrumbs";
+import { mergeMeta } from "~/lib/merge-meta";
+
+export const meta = mergeMeta(({ parentTitle }) => [{ title: `Search for a room - Commercial • ${parentTitle}` }]);
+
+
+export default function searchRoomCommercial() {
```

Comment:
React functional components should start with a capital letter

Reply:
Resolved that in my latest update